keps/sig-node/4639-oci-volume-source/README.md

```diff
@@ -586,8 +586,7 @@ feature cannot be used. Pods using the new `VolumeSource` combined with a not
 supported container runtime version will fail to run on the node, because the
 `Mount.host_path` field is not set for those mounts.
 
-For security reasons, volume mounts should set the [`noexec`] and `ro`
-(read-only) options by default.
+For security reasons, `ro` (read-only) options by default.
```

Comment:
I think generally we should evaluate if we can take the readOnly field from the volume API and apply it, and if we do, we need to determine what `rw` behavior is expected to be

Reply:
I mean this PR focuses on removing noexec, and I'll leave `ro` or `rw` discussion to https://github.com/kubernetes/enhancements/pull/5288 .
Do you mean this change needs discussion about `rw`?